examples/README.md

```diff
@@ -0,0 +1,43 @@
+# LTI Authenticator Examples
+
```

Comment:
Perhaps you could add a sentence here that describes the outcome of following all steps here or similar? I find myself often working through examples to figure out what the example actually is about.

I understand this is an example procedure to deploy JupyterHub locally, configured to authenticate against a LTI v1.1 identity provider, compared to the latest version v1.3?

Reply:
I updated the readme for clarity. Your assumption is almost correct, it doesn't necessarily compare LTI 1.1 with LTI 1.3, rather it provides working configuration examples so users can test different settings with the LTI authenticators. Once we merge #39 then we can include examples for LTI 1.3 as well.